libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Recognizers/AdaptiveRecognizer.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Licensed under the MIT License.
+// Copyright (c) Microsoft Corporation. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using AdaptiveExpressions.Properties;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+
+namespace Microsoft.Bot.Builder.Dialogs.Adaptive.Recognizers
+{
+    /// <summary>
+    /// Recognizer implementation, intended to be a base class for adaptive recognizers.
+    /// </summary>
+    public abstract class AdaptiveRecognizer : Recognizer
```

Comment:
This new base class does 2 things:
* Adds `LogPersonalInformation` flag
* Overrides `FillRecognizerResultTelemetryProperties` method to ensure that the `LogPersonalInformation` flag is accounted for when creating properties to log to telemetry, which the original `Recognizer.FillRecognizerResultTelemetryProperties` method currently fails to do